CitiesHarmony.IPatch/CMPatch.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Mono.Cecil;
+using Mono.Cecil.Cil;
+using Patch.API;
+using System;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+
+namespace CitiesHarmony.API {
+    public class CMPatch : IPatch {
```

Comment:
to use patch loader simply implement IPatch.
Patchloader with scan all dlls for implementations of IPatch and then Executes them